fe/src/App.tsx

```diff
@@ -2,19 +2,20 @@ import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Home from './pages/Home';
 import About from './pages/About';
 import Error from './pages/Error';
-import Header from './components/layouts/Header';
+import Default from './components/layouts/Default';
 
 function App(): JSX.Element {
   return (
     <>
-      <Header />
-      <Router>
-        <Switch>
-          <Route exact path="/" component={Home} />
-          <Route path="/about" component={About} />
-          <Route component={Error} />
-        </Switch>
-      </Router>
+      <Default>
```

Comment:
currently this is fine but in future pr need to think how to implement per page switching.